### ISSUE TITLE
Dashboard Token Listing: EUROe 

### ISSUE THREAD

**Issue by SFServer** [superfluid-finance/assets#153]
Discord Handle of Submitter: juicy#5959

Token Symbol: EUROe

Network: Ethereum Mainnet

Token Address: 0x9a839e70598ef31567006140334908bf507d2cfa

Network Explorer Link: https://etherscan.io/token/0x9a839e70598ef31567006140334908bf507d2cfa

Is a Super Token? true

Address of Underlying Token:  0x820802fa8a99901f52e39acd21177b0be6ee2974

SVG Logo Link: https://dev.euroe.com/img/euroe-token-symbol-svg/euroe-256x256-round.svg

Coingecko or Hardcoded Price?: Coingecko

Coingecko ID: https://www.coingecko.com/en/coins/euroe-stablecoin

Harcoded Price (if Coingecko ID is unavailable): 

Token HEX Color: #243c49

Info on Listing: EUROe is a EU-regulated stablecoin pegged to the Euro and fully backed with bonds & cash.

Please close only after token has been added to resolver!

**Comment by vmichalik** [comment]
closing, this is a duplicate of #142
